app/editor/src/components/form/formpage/styled/FormPage.tsx

```diff
@@ -4,7 +4,7 @@ import { IFormPageProps } from '../FormPage';
 
 export const FormPage = styled.div<IFormPageProps>`
   background-color: white;
-  height: ${(props) => props.height ?? '100%'};
+  min-height: 100%;
```

Comment:
Unrelated to story, but so the `FormPage` will always take up full space of vertical as indicated in mockups